app/assess/data.py

```diff
@@ -110,6 +112,22 @@ def get_round(fund_id: str, round_id: str) -> Round | None:
     return None
 
 
+def get_applications(params: dict) -> List[Application] | None:
```

Comment:
Lovely abstractions, makes it very easy to work with your code.

Reply:
Thanks